pytest_deadfixtures.py

```diff
@@ -69,11 +69,16 @@ def get_fixtures(session):
             loc = getlocation(fixturedef.func, curdir)
             if (fixturedef.argname, loc) in seen:
                 continue
+
             seen.add((fixturedef.argname, loc))
 
             module = fixturedef.func.__module__
-            if not module.startswith("_pytest.") \
-                    and not module.startswith("pytest_"):
+
+            if (
+                not module.startswith("_pytest.") and
+                not module.startswith("pytest_") and
+                not ('virtualenvs' in loc)
+            ):
```

Comment:
@jairhenrique, great work with this feature! do you think this is gonna cover the cases where we have the virtualenv folder inside the project folder? in a small project, I created this method to check it but I'm far from know if there isn't a better approach (and I'd like to know it).

```
    def _is_virtual_env(self, current_folder):
        python = os.path.exists(current_folder + '/bin/python')
        activate = os.path.exists(current_folder + '/bin/activate')
        return python or activate
```

also would be nice to have tests for it!

Reply:
@anapaulagomes I test with local virtualenv and it fails 😞 , but I see a piece of location common between local and global environment: `site-packages`.

Maybe this code solve our problem, but I don't know how to write a test to cover this scenario.

```python
not ('site-packages' in loc)
```

Reply:
Or...

```python
not re.search(r'\/(python|pypy).?\/site-packages\/', loc)
```

Reply:
I forgot to mention that the environment variable `VIRTUAL_ENV` holds the path of the current virtual environment. But it's not reliable since it changes during the execution.

In this [StackOverFlow](https://stackoverflow.com/questions/8427709/how-does-virtualenv-work) question there are few interesting answers about it. That I said, I think the most reliable way to do it it's checking the binaries.

To test it properly, you can write tests where you mock the `os.walk` return, returning regular and possible virtual environment folders. I can help with it but let's see what the other contributors think about this approach.

Reply:
I had a couple ideas on how to make this change, one was like @jairhenrique did, checking for `virtualenv`, `venv`, etc.

Another one was to verify if the current fixture lives below the "tests root path", this would be fairly generic and I think could work well, but have a flaw that when using the plugin in a simple project with the test files living at the root of the project.. :disappointed: 

Checking for `site-packages` looks good to me. As I probably would never want to introspect what's in there, in the same way as I don't test third-party apps.

I would approach testing this just like @anapaulagomes suggested.
I think you could also use the `testdir` fixture and create the necessary directory structure in the tmp folder created for the test.

PS: I'm not at home right now, so I can make a new release only during the weekend.

Reply:
@jllorencetti test added.